rclpy/test/test_init_shutdown.py

```diff
@@ -83,6 +83,17 @@ def func_double_shutdown():
     return False
 
 
+def func_create_node_without_init():
+    import rclpy
+    from rclpy.exceptions import NotInitializedException
+    try:
+        node = rclpy.create_node('foo')
+        assert node  # silence flake
```

Comment:
You can simply not assign the return value of `create_node` to a variable instead.